storm/src/main/storm/mesos/NimbusMesosScheduler.java

```diff
@@ -98,6 +94,7 @@ public void statusUpdate(SchedulerDriver driver, TaskStatus status) {
     if (status.getTaskId().getValue().contains("logviewer")) {
       updateLogviewerState(status);
     }
+
```

Comment:
Please delete newly introduced newlines here at line 97 and at 125, 128, and 134 as well.